e2e/lib/testing.sh

```diff
@@ -21,12 +21,38 @@ function testing_run_test {
     local testfile=$1
 
     local testname=$(testing_get_test_metadata "$testfile" "TEST-NAME")
-    int_start=$(date +%s)
+    local int_start=$(date +%s)
     echo "+++++ $(date): starting $testfile $testname"
     /bin/bash "$testfile"
     echo "+++++ $(date): finished $testfile $testname"
     printf "%s secs\n" "$(($(date +%s) - int_start))"
 
-    echo "Porch Controller logs"
-    kubectl logs deployment/porch-controllers -n porch-system --since "$(($(date +%s) - int_start))s" | sed -e '/PackageVariant/!d;/resources changed/!d'
+    if [[ ${DEBUG:-false} == "true" ]]; then
+        echo "Porch Controller logs"
+        kubectl logs deployment/porch-controllers -n porch-system --since "$(($(date +%s) - int_start))s" | sed -e '/PackageVariant/!d;/resources changed/!d'
+    fi
+}
+
+function testing_run_group {
+    local testdir=$1
+    local tg=$2
+
+    local int_start=$(date +%s)
+    echo "+++++ $(date): starting test group $tg in $testdir"
+    for t in $TESTDIR/$tg-*.sh; do
+        if [[ $t == *-post.sh ]]; then
+            wait
+            testing_run_test "$t"
+        else
+            testing_run_test "$t" &
```

Comment:
We have to keep printing things in the console, otherwise terraform won't be notified.